techmatters/webchat
# Chi 1853 webchat translation set default language studio

## Description

- New feature: Add language to appConfig and make it available on Studio

### Checklist
- [N/A] New tests added



### Related Issues
Fixes CHI-1853

### Verification steps

Language is now available in Studio via `trigger.message.ChannelAttributes.pre_engagement_data.language` in webAttribute of Messaging Flow. 

To test:
 - You will need to add `"language":"{{trigger.message.ChannelAttributes.pre_engagement_data.language}}"` to the webAttribute of the Messaging Flow of whatever helpline that will make use of it.
- Start a webchat conversation, go to Studio > TaskRouter > Task, and select the Task that is related to your conversation. Language should exist in the Attribute JSON.
<img width="1511" alt="Screenshot 2023-04-19 at 12 35 56 PM" src="https://user-images.githubusercontent.com/40429848/233062962-fa2e326e-aa1f-446a-a7d1-b3a21c9cf181.png">
<img width="1511" alt="Screenshot 2023-04-19 at 12 35 47 PM" src="https://user-images.githubusercontent.com/40429848/233063683-25a8e2af-675c-4fa8-a874-551a4f284cf1.png">
